server/rpm/utils/set-version-release

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
```

Comment:
Can't we get rid of this now that we are using Jinja2?

Reply:
Right - will do.